.github/workflows/merge-commits-to-next-branch.yml

```
@@ -0,0 +1,107 @@
+name: Merge commits to next branch
+
+on:
+  schedule:
+    - cron: "0 */2 * * *"
+
+env:
+  COMMIT_INFO: commit-info
+
+jobs:
+  dequeue:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Azure Storage
+        run: npm i @azure/storage-queue
+      - name: Check queue first item
+        uses: actions/github-script@v6
+        with:
+          script: |
+              const { QueueClient, QueueServiceClient } = require("@azure/storage-queue");
+              const connectionString = `${{ secrets.AZURE_STORAGE_CONNECTION_STRING }}`;
+              const queueName = `${{ secrets.AZURE_QUEUE_NAME }}`;
+              const queueServiceClient = QueueServiceClient.fromConnectionString(connectionString);
+              const queueClient = queueServiceClient.getQueueClient(queueName);
+              var receivedMessages = await queueClient.receiveMessages();
+              const message = receivedMessages.receivedMessageItems[0];
+              $COMMIT_INFO = JSON.parse(message.messageText);
+  create-branch:
+    runs-on: ubuntu-latest
+    if: ( $COMMIT_INFO.label == 'queued' )
+    steps:
+      - name: Create branch
+        uses: actions/github-script@v6
+        with:
+          script: |
+            git fetch origin
+            git checkout next
+            git branch $COMMIT_INFO.sha
+            git checkout $COMMIT_INFO.sha
+            git cherry-pick $COMMIT_INFO.sha
+  createPullRequest:
+    runs-on: ubuntu-latest
+    if: ( $COMMIT_INFO.label == 'queued' )
+    steps:
+      - name: Create Pull Request
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const { repo, owner } = context.repo;
+            const result = await github.rest.pulls.create({
+              title: '[Main:Next] Integrate $COMMIT_INFO.sha',
+              $COMMIT_INFO.author,
+              repo,
+              head: $COMMIT_INFO.sha,
+              base: 'next',
+              body: [
+                'This PR is auto-generated by',
+                '[actions/github-script](https://github.com/actions/github-script).'
+              ].join('\n')
+            });
+            github.rest.issues.addLabels({
+              owner,
+              repo,
+              issue_number: result.data.number,
+              labels: ['merge-with-next', 'automated pr']
+            });
+  update-commit-status:
+    runs-on: ubuntu-latest
+    needs: [createPullRequest]
+    if: ( needs.createPullRequest.result == 'success' )
+    steps:
+      - name: Install Azure Queue
+        run: npm i @azure/storage-queue
```

Should you install globally?

```suggestion
        run: npm i -g @azure/storage-queue
```